gulpfile.js

```diff
@@ -3,40 +3,86 @@ var deployDir = 'docs';
 
 var gulp = require('gulp');
 var clean = require('gulp-clean');
+var concat = require('gulp-concat');
 var jshint = require('gulp-jshint');
+var uglify = require('gulp-uglify');
+var babel = require('gulp-babel');
+var inject = require('gulp-inject');
+var minifyHtml = require('gulp-minify-html');
+var minifyCss = require('gulp-clean-css');
+var browserSync = require('browser-sync');
 
-gulp.task('default', ['copy']);
+gulp.task('default', ['lint', 'html']);
 
-gulp.task('copy', ['clean', 'lint'], function() {
-    return gulp.src(src)
-        .pipe(gulp.dest(deployDir));
+gulp.task('build', ['build-html']);
+
+gulp.task('clean', function () {
+  return gulp.src(deployDir)
+      .pipe(clean());
+});
+
+gulp.task('lint', function () {
+  var filesToLint = src.map(function (path) {
+    return path + '/*.js';
+  });
+  filesToLint.push('gulpfile.js');
+
+  return gulp.src(filesToLint)
+      .pipe(jshint({
+        eqeqeq: true,
+        esversion: 6,
+        eqnull: true
+      }))
+      .pipe(jshint.reporter('jshint-stylish'))
+      .pipe(jshint.reporter('fail'));
+});
+
+gulp.task('html', function () {
+  var target = gulp.src('src/index.html');
+  var sources = gulp.src(['src/**/*.js', 'src/**/*.css']);
+
+  return target.pipe(inject(sources, {
+    relative:true
+  }))
+      .pipe(gulp.dest('src'));
```

Comment:
`gulp html` modifies `src/index.html`? It injects all bower_components 'js' and 'css' files regardless of whether or not we explicitly import them?